src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -301,54 +202,12 @@ public void handleList(String listType) {
      * @param addType Typo of person to be added.
      */
     public void handleAddByPopup(String addType) {
-        addCommandPopupWindow = new AddCommandPopupWindow(logic, addType, resultDisplay);
```

Comment:
An alternative is to use multithreading here to wait for command generated by the popup window.
But it is ok to leave it like this.